src/content/docs/agile-handbook/sprint-mechanics/liaisonships.mdx

```diff
@@ -208,39 +152,32 @@ However, a liaison is not the only author on a project. Liaisons should structur
             </td>
             <td>
                 As liaison, you're the expert on both the feature the product team is building, and the docs content (new and existing) that will support that feature. Build an IA that will meet all project needs and scale to the future.
+
+                For an active liaisonship, that means looking across all the docs and ensuring they read well and connect together into a coherent architecture. 
+                
+                If you're a point person, it means building a map in your head of what content is working well and where the opportunities are. You might also bring in smaller tickets into a sprint, or work with your manager if you think we should turn a point person assignment into an active liaisonship.
             </td>
         </tr>
         <tr>
             <td>
                 Write content
             </td>
             <td>
-                Team
-            </td>
-            <td>
-                The liaison writes much of the content for their project, especially the conceptual content like intro docs. But the whole team is expected to swarm and contribute to large projects, with the liaison coordinating that work.
+                Liaison with support from the team
             </td>
-        </tr>
-        <tr>
             <td>
-                Peer edit drafts
-            </td>
-            <td>
-                Liaison
-            </td>
-            <td>
-                When we swarm and have someone else contribute to the project, the liaison peer edits their drafts to ensure consistency with the overall vision.
+                The liaison writes much of the content for their project, especially the conceptual content like intro docs. But they're not the only ones who can contribute! Particularly when things get busy, we expect the whole team to swarm and contribute to large projects, with the liaison coordinating that work.
             </td>
         </tr>
         <tr>
             <td>
-                Coordinate publication
+                Edit content
             </td>
             <td>
-                Liaison
+                Liaison or point person
             </td>
             <td>
-                When the time comes to release (whether that's beta, GA, limited release, or EoL), it's the liaison's job to coordinate with PM, Eng, and Product Marketing to ensure docs go out on time with other deliverables.
+                While many people might edit docs in an area, you're accountable as liaison or point person for the overall quality and structure of the content. That might mean volunteering for peer edit assignments, or tag-teaming with the hero to review an incoming pull request. 
```

Comment:
It could be good to reference (somewhere) the importance of cultivating and maintaining a _voice_ as part of the cohesive experience. Yes, we have a style guide to, well, guide the writers, but drawing attention to what goes into "quality" can empower liaisons to own that area.  Our style guide is a great starting point, but multiple writers can be within the parameters of those guidelines and still have our own unique voices explaining a feature/capability. "Voice" is also a tangible example of what goes into "quality," which is a vague and subjective term. 

A little nitpicky but something that came up for me is how much I should change my own natural voice when writing a doc—it's unclear how much creative control tws have with writing style, etc.

Reply:
I actually want to modify "a little nitpicky," because I do think cultivating a strong, consistent voice within a given area is important for creating trust between reader/user and the feature they're using. 

Based on the rest of the doc, it does seem like there's some flexibility between who is contributing to a given area (e.g., liaisons work with different teams, other writers can swarm and support), such that the liaison should yield when appropriate. But at the very, very least, what makes a quality docs experience can be quite specific and nameable (that idea of empowering your writers to use their voice within style guidelines) without being restrictive/proscriptive (which the word "quality," vague and subjective as it is, can imply).